File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        new = 0

        for i in range(len(nums)):
            if nums[i] == 1:
                count += 1
                if count > new:
                    new = count
            else:
                count = 0

        return new

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {

    public boolean hasDuplicate(int[] nums) {
        Set<Integer> newSet = new HashSet<Integer>();

        for (int n : nums) {
            newSet.add(n);
        }
        if (newSet.size() == nums.length) {
            return false;
        }

        return true;
    }
}